src/utils/color.ts

```diff
@@ -24,3 +25,24 @@ export function getColorEnabled(): boolean {
 
   return !isNoColor
 }
+
+/**
+ * Get whether color change on terminal is enabled or disabled.
+ * If `NO_COLOR` environment variable is set, this function returns `false`.
+ * @see {@link https://no-color.org/}
+ *
+ * @returns {boolean}
+ */
+export async function getColorEnabledAsync(): Promise<boolean> {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const { navigator } = globalThis as any
+
+  const isNoColor =
+    navigator !== undefined && navigator.userAgent === 'Cloudflare-Workers'
+      ? // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        'NO_COLOR' in ((await import('cloudflare:workers')).env ?? {}) // ?? {} is for backward compat
```

Comment:
got this error from this line:

<img width="874" alt="image" src="https://github.com/user-attachments/assets/04d427a8-62e9-46e3-a96a-65f5b30841e2" />

environment: monorepo + nextjs

cc @yusukebe

Reply:
@ysm-dev Can you create an issue?